manifests/profiles/PROFILES.md

```diff
@@ -10,7 +10,7 @@ will provide further instructions. Additionally, please read the RFC requirement
 
 ## Vendor optimized profiles
 
-| K8s Platform         | Vendor        | Lead Maintainer | Lead Maintainer  |
-| -------------------- | ------------- | --------------- | ---------------- |
-| OpenShift            | Red Hat / IBM | @jwendell       | @irisdingbj      |
-| Upstream Kubernetes  | Istio         | Env WG          | discuss.istio.io |
+|Profile Name| K8s Platform         | Vendor        | Lead Maintainer | Lead Maintainer  |
+|------------| -------------------- | ------------- | --------------- | ---------------- |
+|openshift| OpenShift            | Red Hat / IBM | @jwendell       | @irisdingbj      |
+|| Upstream Kubernetes  | Istio         | Env WG          | discuss.istio.io |
```

Comment:
Is discuss.istio.io actually the lead maintainer here?

Reply:
In my opinion: Rename this file to vendor-profiles.md **or, better yet** Drop it entirely and put it in docs website somewhere.

If we decide to keep this file: Drop the upstream kubernetes file, as it is not a vendor.

Reply:
I like the idea of dropping this and adding it directly to the docs. Was surprised to see it here. 

I also ready this as there being a custom vendor profile specifically for Kubernetes but I was unable to find it hence my other question.

Reply:
line 16 is meant to represent the other profiles. The other profiles are meant to run on upstream K8s - and are maintained by the K8s community.

An improved way to represent this table is to show each profile and its maintianer.

I had hoped at some point `profile list` could parse this file and produce support information - hence why it is in repo. I don't see major harm in reproducing the docs in two places if istio.io is desired. I was simply trying to keep it simple 💯

Reply:
Thanks for the clarification @sdake. 


When you say "An improved way to represent this table is to show each profile and its maintianer." Are you referring to what I added in this PR? I think all of that info is already there. 

In my opinion, it would make the most sense if these were listed alongside the other profiles here: https://preliminary.istio.io/latest/docs/setup/additional-setup/config-profiles/. Right now we kind of say "here's our standard installation profiles. Look at the source code for more." And the second additional profile is just a conglomerate reference to the profiles you just came from. For now, it might make sense to have the duplicate content to make it clearer for users. 

Having the list auto-generated off of the profiles list sounds like it might be useful and we have other docs generated off of content from other repositories already, so potentially that content might be able to be used to update the istio.io page?